app/api/core/io/base/json_formatting_behaviour/input.rb

```diff
@@ -23,7 +23,7 @@ def set_model_for_input(model)
     self.model_for_input = model
   end
 
-  def generate_json_to_object_mapping(json_to_attribute)
+  def generate_json_to_object_mapping(json_to_attribute) # rubocop:todo Metrics/CyclomaticComplexity
     code = []
```

Comment:
Just brought some of the rubocop ignores inline. Should keep a cap on the complexity of new code.